From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Fri, 12 Aug 2022 21:25:37 +0200
Subject: [PATCH] libmultipath: convert license of strbuf code to GPL-2.0+

This (partly) fixes the license incompatibility reported in
https://github.com/opensvc/multipath-tools/issues/36

As I'm the only author (except for a trivial spelling fix),
I see no issue with changing the license.

Signed-off-by: Martin Wilck <mwilck@suse.com>
Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 libmultipath/strbuf.c | 2 +-
 libmultipath/strbuf.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libmultipath/strbuf.c b/libmultipath/strbuf.c
index f654594d..e23b65e8 100644
--- a/libmultipath/strbuf.c
+++ b/libmultipath/strbuf.c
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2021 SUSE LLC
- * SPDX-License-Identifier: GPL-2.0-only
+ * SPDX-License-Identifier: GPL-2.0-or-later
  */
 #include <inttypes.h>
 #include <stdint.h>
diff --git a/libmultipath/strbuf.h b/libmultipath/strbuf.h
index 31ab519a..ae863417 100644
--- a/libmultipath/strbuf.h
+++ b/libmultipath/strbuf.h
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2021 SUSE LLC
- * SPDX-License-Identifier: GPL-2.0-only
+ * SPDX-License-Identifier: GPL-2.0-or-later
  */
 #ifndef _STRBUF_H
 #define _STRBUF_H
